docs/deployment-process/execution-containers-for-workers/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Execution containers for workers
+description: How to set a step in your deployment process to run inside a container.
+position: 15
+---
+
+For a [step](/docs/deployment-process/steps/index.md) running on a [worker](/docs/infrastructure/workers/index.md) or on [Octopus server](/docs/infrastructure/workers/built-in-worker.md) that you define in your [deployment processes](/docs/deployment-process/index.md), you can select a container for the step's execution. 
+
+## Minimum requirements for your custom image to run as a worker
+The worker will need Docker installed. 
```

Comment:
We should link here to the requirements for a worker, and then say "all of those, plus docker"

Reply:
@emilol I assume those requirements will be on DockerHub right?

Reply:
I don't think so. The minimum requirements for a worker relate to the host machine that `docker run` will get called from, not the image itself. You can check out the WorkerTools repo for the readme structure we've landed on for DockerHub